docs/configuring-playbook-own-webserver.md

```diff
@@ -26,4 +26,75 @@ matrix_nginx_proxy_enabled: false
 
 - ensure that the `/.well-known/acme-challenge` location for each "port=80 vhost" gets proxied to `http://localhost:2402` (controlled by `matrix_ssl_certbot_standalone_http_port`) for automated SSL renewal to work
 
-- ensure that you restart/reload your webserver once in a while, so that renewed SSL certificates would take effect (once a month should be enough)
\ No newline at end of file
+- ensure that you restart/reload your webserver once in a while, so that renewed SSL certificates would take effect (once a month should be enough)
+
+**Apache2 sample configuration file**
+
+1. Create a new apache configuration file named 000-matrix-ssl.conf and enable it. Make certain to replace DOMAIN/SSL values with the correct ones for your server.
+
+       # Auto redirect http to https
+       <VirtualHost *:80>
+           ServerName matrix.DOMAIN
+           Redirect permanent / https://matrix.DOMAIN/
+       </VirtualHost>
+
+       <VirtualHost *:443>
+           ServerName matrix.DOMAIN
+
+           SSLEngine On
+           SSLCertificateFile /etc/letsencrypt/live/DOMAIN/fullchain.pem
```

Comment:
Maybe this can be set to:

```
SSLCertificateFile /matrix/ssl/config/live/matrix.DOMAIN/fullchain.pem
SSLCertificateKeyFile /matrix/ssl/config/live/matrix.DOMAIN/privkey.pem
```

for a better default.